app-tasks/rf/src/rf/uploads/sentinel2/settings.py

```diff
@@ -5,6 +5,43 @@
 # Public Organization UUID in Raster Foundry
 organization = 'dfac6307-b5ef-43f7-beda-b9f208bb7726'
 
+# Sentinel 2 bands
+# Bands 2, 3, 4, and 8 are at the 10m resolution. Bands 5, 6, 7, 8a, 11, and 12
+# are at the 20m resolution. Bands 1, 9, and 10 are at the 60m resolution.
+# Bands are:
+# 1: Coastal aerosol
+# 2: Blue
+# 3: Green
+# 4: Red
+# 5: Narrow NIR 1
+# 6: Narrow NIR 2
+# 7: Narrow NIR 3
+# 8: NIR
+# 8a: Narrow NIR 4
+# 9: Water vapor
+# 10: Cirrus
+# 11: SWIR 1
+# 12: SWIR 2
+
+band_lookup = {
```

Comment:
The Landsat settings file calls this `resolution_band_lookup`; it'd be good to make the two consistent.